src/structure/Header.tsx

```diff
@@ -85,8 +88,9 @@ function Header({ isDarkMode, darkModeToggle }) {
 
   // Get Location
   const location = useLocation();
+  const isSnapPage = location?.pathname === '/snap';
 
-  // const [snapInstalled, setSnapInstalled] =  useState(false);
+  const showRewardsLinkInMobile = useDeviceWidthCheck(600);
```

Comment:
Instead of this, let's use box display property to show and hide the rewards circle container in different screen sizes, as we did in Text.